frontend/ui/wikipedia.lua

```diff
@@ -352,8 +352,8 @@ function Wikipedia:createEpub(epub_path, page, lang, with_images, progress_callb
             local imgid = string.format("img%05d", imagenum)
             local imgpath = string.format("images/%s.%s", imgid, ext)
             local mimetype = ext_to_mimetype[ext] or ""
-            local width = img_tag:match([[width="([^"]*)"]])
-            local height = img_tag:match([[height="([^"]*)"]])
+            local width = tonumber(img_tag:match([[width="([^"]*)"]]))
```

Comment:
Actually I had a bit of a brainfart. You could just say `local width = tonumber(img_tag:match([[width="([^"]*)"]])) or 0` and then the check down in cover_imgid could be much simpler.

(Incidentally, why does it need to be at least 50?)

Reply:
I use this width to rewrite the HTML, so with nil, it now generates:
`<img src="images/img00001.png?fcc27" style="width: nilpx; height: 4px" alt=""/>` :)
which is ok for crengine. I think having width: 0px would may be make the image really 0px wide and invisible.
Well, OK, I should then probably rewrite the building of this HTML snipet to not include width or height if they are nil...

50, because we are choosing an image among those included in the wikipedia page to act as the EPUB cover image. There's often small icons among the first images, and it looked to me any image > 50 was actually a content image, and usable as a cover (and width < height, cause we cant an image in portrait mode)

Reply:
Makes sense. So stick an if width/height where relevant. 👍